Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -186,73 +184,6 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         self.concurrencySemaphore = DispatchSemaphore(value: Concurrency.maxOperations)
     }
 
-    // deprecated 8/2021
```

Comment:
@neonichu could/should we do a separate PR to remove all the deprecated API, get that merged than do this PR on top?

Reply:
Not sure, if you think it's valuable to split it out, I can definitely do that.

Reply:
feels like the right order of things, also allows us to back out one without backing out the other

Reply:
https://github.com/apple/swift-package-manager/pull/4256

Reply:
That one removes all deprecated API that was deprecated 08/2021